dashboard/app/views/pd/application/teacher1920_application_mailer/principal_approval_completed_partner.html.haml

```diff
@@ -14,7 +14,7 @@
   Best,
 
 %p
-  Anthonette Peña
+  Liz Gauthier
```

Comment:
Adjusts the "from" header on three emails:

- `principal_approval_completed_partner__with_partner`
- `principal_approval_completed_partner__with_partner_no_contact`
- `principal_approval_completed_partner__without_partner`

(Same ones we adjusted the "from" header for, above.)